docker/config/l1-node/el/genesis.json

```diff
@@ -998,5 +998,5 @@
   "nonce": "0x1234",
   "mixhash": "0x0000000000000000000000000000000000000000000000000000000000000000",
   "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
-  "timestamp": "1708874895"
+  "timestamp": "0x0"
```

Comment:
As suggested [here](https://besu.hyperledger.org/public-networks/reference/genesis-items#:~:text=timestamp,the%20genesis%20file.) from besu official doc